public/index.php

```diff
@@ -1,7 +1,36 @@
 <?php
+
+namespace GanttDashboard;
+
 use Phalcon\Di\FactoryDefault;
 
-error_reporting(E_ALL);
+/**
+ * Application environment
+ */
+$environment = isset($_SERVER['APP_ENV']) ? strtolower($_SERVER['APP_ENV']) : 'development';
+
+/**
+ * Error reporting
+ */
+if (false === in_array($environment, ['testing', 'production', 'development'])) {
+    header('HTTP/1.1 503 Service Unavailable.', true, 503);
+    echo 'The application environment is not set correctly.';
+    exit(1);
+}
+
+$errorReporting = E_ALL;
+$displayErrorsParam  = 1;
```

Comment:
avoid using variables like: $filterArray, $userObject, $urlString, $myParam...
why nit $iniDisplayErrors, $iniLogErrors.... as this is what they do... they overwrite in php.ini